tests/acceptance/blocking_adapter_test.py

```diff
@@ -476,13 +469,13 @@ def test(self):
         self.assertIsInstance(method, pika.spec.Basic.GetOk)
         self.assertEqual(method.delivery_tag, 1)
         self.assertFalse(method.redelivered)
-        self.assertEqual(method.exchange, _p3_as_bytes(''))
-        self.assertEqual(method.routing_key, q_name)
+        self.assertEqual(method.exchange, as_bytes(''))
+        self.assertEqual(method.routing_key, as_bytes(q_name))
```

Comment:
Thanks @everilae: I was looking for `shortstr` in that doc, and came up empty-handed. this is perfect! Since shortstr are UTF-8 and objects such as queue names, exchange names, routing keys are defined as shortstr, pika should accept strings for any shortstr fields from the user and return unicode strings for shortsr fields back to the user. There is not need for the bytes there at all, right?